components/markets/market-card/index.tsx

```diff
@@ -70,55 +155,112 @@ const MarketCard = ({
   question,
   creation,
   outcomes,
-  prediction,
+  marketType,
   volume,
   baseAsset,
   width,
+  tags,
+  endDate,
+  status,
   className = "",
 }: MarketCardProps) => {
-  const [showDetailsOverlay, setShowDetailsOverlay] = useState<boolean>(false);
+  const { data: spotPrices } = useMarketSpotPrices(marketId);
```

Comment:
You shouldn't need to do this as we were already displaying the prediction value so the underlying prices must be getting fetched from somewhere. It will vary depending on where the data is coming from, I can help you work through it if you like

Reply:
Thanks for the direction here; I was able to sort it out!